plugins/inventory/k8s.py

```diff
@@ -98,27 +98,27 @@
     - "PyYAML >= 3.11"
 """
 
-EXAMPLES = """
+EXAMPLES = r"""
 # File must be named k8s.yaml or k8s.yml
 
-# Authenticate with token, and return all pods and services for all namespaces
-plugin: kubernetes.core.k8s
-connections:
-  - host: https://192.168.64.4:8443
-    api_key: xxxxxxxxxxxxxxxx
-    validate_certs: false
-
-# Use default config (~/.kube/config) file and active context, and return objects for a specific namespace
-plugin: kubernetes.core.k8s
-connections:
-  - namespaces:
+- name: Authenticate with token, and return all pods and services for all namespaces
+  plugin: kubernetes.core.k8s
+  connections:
+    - host: https://192.168.64.4:8443
+      api_key: xxxxxxxxxxxxxxxx
+      validate_certs: false
+
+- name: Use default config (~/.kube/config) file and active context, and return objects for a specific namespace
+  plugin: kubernetes.core.k8s
+  connections:
+    - namespaces:
     - testing
```

Comment:
This should be intended probably?!

Reply:
Ansible-lint version 6.21 examines the indentation in the EXAMPLES block. While it could be skipped, I chose not to do so because the issue was observed in only a few files.

Reply:
I mean - testing should be indented. It should be a namespace, or am I wrong.

Reply:
Yeah, @alinabuzachis is correct. I just pushed a fix.